src/AggregateFunctions/AggregateFunctionUniq.h

```diff
@@ -124,6 +125,26 @@ struct AggregateFunctionUniqExactData<String>
 };
 
 
+/// uniqThetaSketch
+#if USE_DATASKETCHES
+struct AggregateFunctionUniqThetaSketchData
+{
+    using Set = ThetaSketchData<UInt64>;
+    Set set;
+
+    static String getName() { return "uniqThetaSketch"; }
+};
+
+/// For a function that takes multiple arguments. Such a function pre-hashes them in advance, so TrivialHash is used here.
+struct AggregateFunctionUniqThetaSketchDataForVariadic
```

Comment:
This struct is completely the same as `AggregateFunctionUniqThetaSketchData`, why do we need it?

Reply:
Ah...we do not need it. It's my carelessness. Comment addressed, PTAL~